src/main/java/it/gov/pagopa/pu/send/connector/client/UploadClient.java

```diff
@@ -0,0 +1,8 @@
+package it.gov.pagopa.pu.send.connector.client;
+
+import it.gov.pagopa.pu.send.dto.DocumentDTO;
+import java.util.Optional;
+
+public interface UploadClient {
+  Optional<String> uploadFileToS3(DocumentDTO documentDTO, byte[] fileBytes);
```

Comment:
just upload, let's not expose the integration detail
```suggestion
  Optional<String> uploadFile(DocumentDTO documentDTO, byte[] fileBytes);
```